gson/src/test/java/com/google/gson/FieldNamingPolicyTest.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.gson;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.assertEquals;
+
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+
+/**
+ * Unit test for the {@link FieldNamingPolicy} class
+ * 
+ * @author David Betancourt
+ *
+ */
+@RunWith(JUnitParamsRunner.class)
+public class FieldNamingPolicyTest {
+
+  private Object[] getUpperCaseFirstLetterParams() {
+    return new Object[] { getUpperCaseFirstLetterParamArgs("someFieldName", "SomeFieldName"),
```

Comment:
might be more legible if every call to `getUpperCaseFirstLetterParamArgs` was on its own line?